keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/KeycloakClient.kt

```diff
@@ -288,4 +289,25 @@ interface KeycloakClient {
     @RequestLine("GET /admin/realms/{realm}/groups/{group-id}/role-mappings/realm/composite")
     fun groupRoles(@Param("realm") realm: String, @Param("group-id") id: UUID): List<RoleListItem>
 
+    @RequestLine("GET /admin/realms/{realm}/clients/{client-id}/scope-mappings/realm")
+    fun realmRoleScopeMappings(@Param("realm") realm: String, @Param("client-id") id: UUID): List<RoleScopeMapping>
```

Comment:
Maybe we can use a clearer method name here like: clientRealmRoleScopeMappings?

Reply:
done